views/reducers/session.jsx

```diff
@@ -31,6 +41,8 @@ function session(state = {}, action) {
   switch (action.type) {
     case PROCESS_USER:
     case SIGNED_IN:
+    case ERROR:
+    case RESET_AUTH_ERROR:
     case SIGNED_OUT:
       return Object.assign({}, state, user(state.session, action));
```

Comment:
I debugged on master, after sign in action is done, state.session is still null, so it will use default values.